applications/nrf_desktop/src/events/config_event.h

```diff
@@ -35,7 +35,8 @@ extern "C" {
 	X(TIMEOUT)			\
 	X(REJECT)			\
 	X(WRITE_FAIL)			\
-	X(DISCONNECTED)
+	X(DISCONNECTED)			\
+	X(TRANSPORT_DISABLED)
```

Comment:
can't we reuse disconnected to leave the protocol as is?

Reply:
Discussed offline. We will simply use disconnected status here to avoid modifying protocol.